autogpt_platform/backend/backend/blocks/basic.py

```diff
@@ -2,10 +2,10 @@
 from typing import Any, List
 
 from jinja2 import BaseLoader, Environment
-from pydantic import Field
 
 from backend.data.block import Block, BlockCategory, BlockOutput, BlockSchema, BlockType
 from backend.data.model import SchemaField
+from backend.data.model import SchemaField as Field
```

Comment:
I'd prefer having a bigger diff here doing a find and replace on Field with SchemaField. 

Overriding Field will probably lead to confusion down the line

Reply:
Okay updated.